### DISCUSSION TITLE
Release to address AddFluentValidation() warnings planned soon?

### DISCUSSION THREAD

**Discussion by hsandhar** [jasontaylordev/CleanArchitecture#818] [Q&A] [answered]
I noticed @jasontaylordev pushed [changes](https://github.com/jasontaylordev/CleanArchitecture/commit/73ef333450cbc31e3812c82151d33cbc14270b01) that addresses the [AddFluentValidation()](https://github.com/FluentValidation/FluentValidation/issues/1965) warnings.

I was just wondering if there's a release planned in the near future so this gets pushed to the template as well.

**Comment by jasontaylordev** [comment, marked as answer]
I pushed a new version of the template this morning; <https://www.nuget.org/packages/Clean.Architecture.Solution.Template>.
